src/graphql/mongoose_graphql_cowboy_handler.erl

```diff
@@ -6,38 +6,61 @@
 %% @end
 -module(mongoose_graphql_cowboy_handler).
 
+-behaviour(mongoose_http_handler).
 -behavior(cowboy_rest).
 
+%% mongoose_http_handler callbacks
+-export([config_spec/0]).
+
+%% config processing callbacks
+-export([process_config/1]).
+
 %% Cowboy Handler Interface
 -export([init/2]).
 
 %% REST callbacks
--export([
-    allowed_methods/2,
-    resource_exists/2,
-    content_types_provided/2,
-    content_types_accepted/2,
-    charsets_provided/2,
-    is_authorized/2
-]).
+-export([allowed_methods/2,
+         resource_exists/2,
+         content_types_provided/2,
+         content_types_accepted/2,
+         charsets_provided/2,
+         is_authorized/2]).
 
 %% Data input/output callbacks
--export([
-    from_json/2,
-    to_json/2,
-    to_html/2
-]).
-
--ignore_xref([cowboy_router_paths/2, from_json/2, to_html/2, to_json/2]).
+-export([from_json/2,
+         to_json/2,
+         to_html/2]).
+
+-ignore_xref([from_json/2, to_html/2, to_json/2]).
+
+-include("mongoose_config_spec.hrl").
+
+%% mongoose_http_handler callbacks
+
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{items = #{<<"username">> => #option{type = binary},
+                       <<"password">> => #option{type = binary},
+                       <<"schema_endpoint">> => #option{type = binary}},
```

Comment:
I think there are only a few options here and we shouldn't allow an arbitrary binary, right? Using `enum` looks like a better option to me, because it would be validated. Another question: would it make sense to make the endpoint an atom?